### PR TITLE
Update Russian translation

### PR DESCRIPTION
**Pull request by Alexey71**
_If your pull request is a translation update, then the title of this pull must contains 'string' or 'translation'_

### CODE REVIEW

**Comment by MlgmXyysd** [comment]
Do NOT translate "translatable=false" strings

File: app/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
 
+    <!-- Do NOT translate these -->
+    <string name="download_progress" translatable="false">%1$,d / %2$,d кб</string>
```
Comment (MlgmXyysd) [inline app/src/main/res/values-ru/strings.xml:5]: Changes required

**Review by MlgmXyysd — CHANGES_REQUESTED** [review]

**Comment by Alexey71** [comment]
Can you turn on the translation ?.  Since this line requires translation.

File: app/src/main/res/values-ru/strings.xml (hunk 1 above)
Comment (Alexey71) [reply app/src/main/res/values-ru/strings.xml:5]: This line needs translation.  Can you remove not translatable?

Comment (MlgmXyysd) [reply app/src/main/res/values-ru/strings.xml:5]: > This line needs translation. Can you remove not translatable?

Sorry, it's my fault. It's fixed

**Pull request closed without merging**